GTG/core/xml.py

```diff
@@ -0,0 +1,316 @@
+# -----------------------------------------------------------------------------
+# Getting Things GNOME! - a personal organizer for the GNOME desktop
+# Copyright (c) 2008-2013 - Lionel Dricot & Bertrand Rousseau
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+# -----------------------------------------------------------------------------
+
+import os
+import shutil
+import xml.sax.saxutils as saxutils
+from datetime import datetime
+from GTG.core.dates import Date
+from GTG.core.logger import log
+
+from lxml import etree
+
+
+# Total amount of backups
+BACKUPS = 7
+
+# Information on whether a backup was used
+backup_used = {}
+
+
+def task_from_element(task, element: etree.Element):
```

Comment:
If you wanted to be technical, `element: etree._Element` but I believe this should work just fine.

Reply:
Nah, the IDE picks it up fine